ProcessLib/HT/Tests.cmake

```diff
@@ -448,3 +448,24 @@ AddTest(
     ThermalConvection_pcs_0_ts_1_t_0.000000_expected.vtu ThermalConvection_pcs_0_ts_1_t_0.000000.vtu darcy_velocity darcy_velocity 1e-8 1e-13
     VIS ThermalConvection_pcs_0_ts_1_t_0.000000.vtu
 )
+
+if(NOT OGS_USE_MPI AND BUILD_TESTING AND Python_FOUND)
+    add_custom_target(generate_invalid_project_files_ht
+        ${Python_EXECUTABLE}
+        ${PROJECT_SOURCE_DIR}/ThirdParty/ogs6py/generateInvalidMediaForHT.py
+                                                generateInvalidMediaForHT.py
+        WORKING_DIRECTORY ${Data_SOURCE_DIR}/Parabolic/HT/InvalidProjectFiles/)
+    file(GLOB HT_INVALID_PRJ_FILES ${Data_SOURCE_DIR}/Parabolic/HT/InvalidProjectFiles/*.prj)
+    foreach(ht_invalid_prj_file ${HT_INVALID_PRJ_FILES})
+        string(REPLACE ${Data_SOURCE_DIR}/Parabolic/HT/InvalidProjectFiles/HT "invalid" ht_invalid_prj_file_short ${ht_invalid_prj_file})
```

Comment:
Shouldn't it be the binary dir? Polluting source dir is not nice ;)

Reply:
Will change this tomorrow. Please don't merge.